librz/util/sdb/src/array.c

```diff
@@ -60,16 +60,21 @@ static inline int int_cmp(const void *a, const void *b) {
 	return 0;
 }
 
-RZ_API ut64 sdb_array_get_num(Sdb *s, const char *key, int idx) {
+RZ_API ut64 sdb_array_get_num(Sdb *s, const char *key, int idx, int* err) {
```

Comment:
instead of doing this, return `bool`.

```suggestion
RZ_API bool sdb_array_get_num(Sdb *s, const char *key, int idx, ut64* out_number) {
```

Reply:
in this way you can know if the function errored or not.

Reply:
okay, I'll fix this.